src/containers/stage.jsx

```diff
@@ -32,10 +33,12 @@ class Stage extends React.Component {
         this.attachRectEvents();
         this.attachMouseEvents(this.canvas);
         this.updateRect();
-        this.renderer = new Renderer(this.canvas);
-        this.props.vm.attachRenderer(this.renderer);
         this.audioEngine = new AudioEngine();
         this.props.vm.attachAudioEngine(this.audioEngine);
+        this.renderer = new Renderer(this.canvas);
```

Comment:
The squirrel shows up if you're running scratch-render before LLK/scratch-render#122
The ordering of audio/renderer/storage is just alphabetical, no specific reason.